### DISCUSSION TITLE
Performance comparison of Matplot++ vs matplotlib or other plotting libraries

### DISCUSSION THREAD

**Discussion by alda30** [alandefreitas/matplotplusplus#438] [General]
I recently started looking for a fast plotting library and discovered matplot++. It was a delightful find, especially since I have prior experience with matplotlib. I’m genuinely impressed by the amazing work the community has done with matplot++—kudos to everyone involved!

That said, I haven’t come across any articles or posts discussing its performance compared to matplotlib. I’m curious about how switching to matplot++ might impact the speed of plotting. Does anyone have insights or experience they could share? I’d really appreciate it!

**Comment by alandefreitas** [comment]
It's fast in the sense that you can manipulate the data in C++. The plotting is not that fast because it delegates the work to Gnuplot, which is not fast, especially if you need to refresh it constantly.
